src/Features/CSharpTest/UseVarTestExtensions.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if false
```

Comment:
What's going on here?

Reply:
Not needed on this form, and can be removed in followup.